internal/tmpl/tmpl.go

```diff
@@ -27,6 +27,7 @@ type Fields map[string]interface{}
 const (
 	// general keys.
 	projectName     = "ProjectName"
+	dist            = "Dist"
```

Comment:
I think this is not needed... our internal code can use `ctx.Config.Dist` and user-configs should be scoped to dist already...

Reply:
Circling back to this --I added `Dist` as an option to allow for tool arguments that have to deal with outputting files to allow them to specify the dist directory explicitly:
```
...
-
  Cmd: my-tool
  Args:
    - --file
    - "{{ .Dist }}/some.sbom"
...
```

Reply:
After more thought I decided to go the route of changing the cwd of the SBOM tool to `Dist` and removing `Dist` from templating. This way all cataloging tools are oriented towards what goreleaser generates instead of another location. An additional pro is that it makes it harder to misuse the `$sbom#` variables (you don't need to remember to prefix `{{ .Dist }}/` to all paths). 

The downside of this is that if tooling depends on configuration in the root of the repo this will now need to be explicitly passed in as an option.

Reply:
yes, that was my idea as well!

sorry it took so long to reply, crazy week :P